lms/models/users.py

```diff
@@ -26,3 +29,11 @@ def auth(username: str, password: str) -> User:
     elif user.role.is_unverified:
         raise ForbiddenPermission(_('עליך לאשר את מייל האימות'), 403)
     return user
+
+
+def generate_user_token(user: User) -> str:
+    return SERIALIZER.dumps(user.mail_address, salt=retrieve_salt(user))
+
+
+def generate_session_token(username: str, password: str) -> str:
+    return SERIALIZER.dumps([username, password])
```

Comment:
That's the hashed password, right? ^^"
(If not - we should use the hashed one because this potentially weakens our security in case someone hacked the database)

Reply:
Changed it to be uuid